### PR TITLE
Shipping information

### PR DESCRIPTION
**Pull request by alograg**
Add shiping config for checkouts

### CODE REVIEW

File: src/Checkout.vue [hunk 1]
```diff
@@ -52,6 +52,11 @@ export default {
       type: String,
       default: undefined
     },
+    shippingIn: {
```
Comment (jofftiquez) [inline src/Checkout.vue]: Make this `shippingAddressCollection` instead.

File: src/Checkout.vue [hunk 2]
```diff
@@ -52,6 +52,11 @@ export default {
...
+      type: Array,
+      default: [],
+      validator: (value) => !['AS', 'CX', 'CC', 'CU', 'HM', 'IR', 'KP', 'MH', 'FM', 'NF', 'MP', 'PW', 'SD', 'SY', 'UM', 'VI'].includes(value)
```
Comment (jofftiquez) [inline src/Checkout.vue]: Adjust the validator. Put the array of allowedCountries in the `constants.js` see `SUPPORTED_LANGS` implementation.

File: src/Checkout.vue [hunk 3]
```diff
@@ -71,6 +76,7 @@ export default {
               clientReferenceId: this.clientReferenceId,
               customerEmail: this.customerEmail,
               items: this.items,
+              shippingAddressCollection: {allowedCountries:this.shipping},
```
Comment (jofftiquez) [inline src/Checkout.vue]: Make this `shippingAddressCollection: this.shippingAddressCollection`.

**Comment by jofftiquez** [comment]
Hi @alograg have you checked the review change request? Thanks.

**Comment by jofftiquez** [comment]
Any updates on this? @alograg

**Comment by alograg** [comment]
> Any updates on this? @alograg

I wil work on it the weekend

**Comment by jofftiquez** [comment]
Hello @alograg any updates on this? Thanks.

**Pull request closed without merging**